### PR TITLE
fix: widget sizing

### PR DESCRIPTION
**Pull request by zzmp**

### CODE REVIEW

File: src/lib/components/Widget.tsx [hunk 1]
```diff
@@ -23,14 +23,15 @@ const WidgetWrapper = styled.div<{ width?: number | string }>`
...
+  box-sizing: border-box;
   color: ${({ theme }) => theme.primary};
   display: flex;
   flex-direction: column;
   font-feature-settings: 'ss01' on, 'ss02' on, 'cv01' on, 'cv03' on;
   font-size: 16px;
   font-smooth: always;
   font-variant: none;
-  height: 348px;
+  height: 356px;
```
Comment (ianlapham) [inline src/lib/components/Widget.tsx:34]: is there any way to test this within an integration? i remember changing this didnt expose errors in cosmos but it did when used within other apps 

it affected the bottom amount of padding

**Review by ianlapham — COMMENTED** [review]
Wondering if we can test change within an integration

**Review by JFrankfurt — APPROVED** [review]
🤷‍♀️

File: src/lib/components/Widget.tsx (hunk 1 above)
Comment (zzmp) [reply src/lib/components/Widget.tsx:34]: I manually tested this with widgets-demo to come up with this number.

**Pull request merged**